docs/content/users/install/ddev-installation.md

```diff
@@ -154,8 +154,8 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
 
     * Execute DDEV commands inside WSL2.   
       You’ll want to run DDEV commands inside Ubuntu, for example, and never on the Windows side in PowerShell or Git Bash.
-    * Projects should live in the Linux filesystem.  
-      WSL2’s Linux filesystem (e.g. `/home/<your_username>`) is much faster, so keep your projects there and **not** in the slower Windows filesystem (`/mnt/c`).
+    * Projects should live in the home directory of the Linux filesystem.  
+      WSL2’s Linux filesystem (e.g. `/home/<your_username>`) is much faster and has proper permissions, so keep your projects there and **not** in the slower Windows filesystem (`/mnt/c`).
```

Comment:
Not English as First Language, but I'd say "under"? "in the home directory" suggest me it needs to be /home/myuser/project-name, while /home/myuser/dir1/dir2/project-name is valid.

```suggestion
    * Projects should live under the home directory of the Linux filesystem.  
      WSL2’s Linux filesystem (e.g. `/home/<your_username>`) is much faster and has proper permissions, so keep your projects there and **not** in the slower Windows filesystem (`/mnt/c`).
```